__tests__/app.test.js

```diff
@@ -23,4 +23,21 @@ describe("/api/categories", () => {
   });
 });
 
+describe("/api/reviews/:review_id", () => {
+  it("responds with a status code of 200", () => {
+    return request(app).get("/api/reviews/3").expect(200);
+  });
+  it("responds with the review object specified in the params", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then((response) => {
+        const review_object = response.body.rows[0]

```

Comment:
the ticket requests an object be returned, but this returns an array; refactor please